packages/integration-sdk-cli/src/generator/template/CODEOWNERS.hbs

```diff
@@ -0,0 +1,3 @@
+* @jupiterone/integrations-staff @jupiterone/integrations-contractors
```

Comment:
Based on recent GitHub team changes, I think we'll want to switch this to just be @jupiterone/integrations instead of the two distinct sub-teams.